core/commands/show_commit.py

```diff
@@ -28,3 +33,13 @@ def run(self, edit):
         commit_hash = self.view.settings().get("git_savvy.show_commit_view.commit")
         content = self.git("show","--no-color", commit_hash)
         self.view.run_command("gs_replace_view_text", {"text": content, "nuke_cursors": True})
+
+class GsShowCommitFile(TextCommand, GitCommand):
+
+    def run(self, edit):
+        commit_hash = self.view.settings().get("git_savvy.show_commit_view.commit")
+        filepath = self.view.settings().get("git_savvy.show_commit_view.filepath")
+        repopath = self.view.settings().get("git_savvy.repo_path")
+        file = filepath[len(repopath)+1:]
+        content = self.git("show", commit_hash + ':' + file)
```

Comment:
Moved function statements into run_async. Call run_async after a timeout of 0 ms